### PR TITLE
Brian's Views 

### PR DESCRIPTION
**Pull request by brian-jw-park**
# *Name of the PR*

## :recycle: Current situation & Problem
No view for Brian 

## :bulb: Proposed solution
Now has view for Brian 

## :gear: Release Notes 
Included maps and colored buttons. 

## :heavy_plus_sign: Additional Information
*Provide some additional information if possible*

### Related PRs
*Reference the related PRs*

### Testing
*Are there tests included? If yes, which situations are tested, and which corner cases are missing?*

### Reviewer Nudging
*Where should the reviewer start? Where is a good entry point?*

### Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md):
- [ ] I agree to follow the [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: CS342MemberViews/Sources/BrianPark/BrianPark.swift [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            VStack(spacing: 42) {
+                ProfileView()
+//                ProfileView(profileImage: Bundle.module.image(fromFileNamed: "BrianPark"))
+                DescriptionView()
+                Text("Most recently from:")
+                    .font(.title3)
+                    .padding()
+                MapView()
+                    .frame(height: 300)
+                    .offset(y:-50)
```
Comment (vishnuravi) [inline CS342MemberViews/Sources/BrianPark/BrianPark.swift]: Swiftlint is throwing an error here because the colon should have a single space after it.

```suggestion
                    .offset(y: -50)
```

File: CS342MemberViews/Sources/BrianPark/BrianPark.swift [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                Text("Some of my socials:")
+                    .font(.title3)
+                HStack(spacing: 30) {
+                    LinkedIn()
+                    Instagram()
+                    Github()
+                }
+
```
Comment (vishnuravi) [inline CS342MemberViews/Sources/BrianPark/BrianPark.swift]: SwiftLint is throwing an error here because there is an extra line of vertical whitespace before the closing brace.

```suggestion
```

File: CS342MemberViews/Sources/BrianPark/Github.swift [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+import SwiftUI
+
+
+struct Github: View {
+    @Environment(\.openURL) var openURL
+
+
+    var body: some View {
+        if let GitURL {
```
Comment (vishnuravi) [inline CS342MemberViews/Sources/BrianPark/Github.swift]: The variable name should start with a lowercase letter.

```suggestion
        if let gitURL {
```

File: CS342MemberViews/Sources/BrianPark/Github.swift [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+                    .frame(maxWidth: .infinity, idealHeight: 50)
+                    .bold()
+            }
+                .buttonStyle(.bordered)
+                .tint(.green)
+        }
+    }
+
+
+    private var GitURL: URL? {
```
Comment (vishnuravi) [inline CS342MemberViews/Sources/BrianPark/Github.swift]: SwiftLint throws an error here because the variable name should start with a lowercase letter.

```suggestion
    private var gitURL: URL? {
```

File: CS342MemberViews/Sources/BrianPark/Instagram.swift [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+import SwiftUI
+
+
+struct Instagram: View {
+    @Environment(\.openURL) var openURL
+
+
+    var body: some View {
+        if let InstaURL {
```
Comment (vishnuravi) [inline CS342MemberViews/Sources/BrianPark/Instagram.swift]:
```suggestion
        if let instaURL {
```

File: CS342MemberViews/Sources/BrianPark/Instagram.swift [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+                    .frame(maxWidth: .infinity, idealHeight: 50)
+                    .bold()
+            }
+                .buttonStyle(.bordered)
+                .tint(.black)
+        }
+    }
+
+
+    private var InstaURL: URL? {
```
Comment (vishnuravi) [inline CS342MemberViews/Sources/BrianPark/Instagram.swift]:
```suggestion
    private var instaURL: URL? {
```

File: CS342MemberViews/Sources/BrianPark/MapView.swift [hunk 7]
```diff
@@ -0,0 +1,26 @@
+//
+//  File.swift
+//  
+//
+//  Created by Jeongwoo Park on 2023/01/24.
+//
+
+import SwiftUI
+import MapKit
```
Comment (vishnuravi) [inline CS342MemberViews/Sources/BrianPark/MapView.swift]:
```suggestion
import MapKit
import SwiftUI
```

File: CS342MemberViews/Sources/BrianPark/BrianPark.swift [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import InstructorViews
+import SwiftUI
+import MapKit
```
Comment (vishnuravi) [inline CS342MemberViews/Sources/BrianPark/BrianPark.swift]: SwiftLint is throwing an error here because imports should be sorted in alphabetical order based on the name of the package.

```suggestion
import InstructorViews
import MapKit
import SwiftUI
```

**Review by vishnuravi — CHANGES_REQUESTED** [review]
Hey @brian-jw-park! This assignment looks great! Nice job! 🚀 I like how you used MapKit and added multiple social buttons.

There are a few minor issues with the code style that are preventing the automated checks from passing. I've marked them up for you.

It also looks like you are missing REUSE headers on some of the files you added, including `MapView` and `DescriptionView`.

It would also be great if we could add a UI test for the view you created in `StudentsUITests`. If you need help writing the test, just let us know!

Once you are able to fix these the checks will run again, and we can merge your branch into main after they pass.

Thanks!

File: CS342MemberViews/Sources/BrianPark/MapView.swift [hunk 9]
```diff
@@ -0,0 +1,26 @@
+//
+//  File.swift
+//  
+//
+//  Created by Jeongwoo Park on 2023/01/24.
+//
```
Comment (vishnuravi) [inline CS342MemberViews/Sources/BrianPark/MapView.swift]: There should be a REUSE header here, such as:

```swift
//
// This source file is part of the CS342 2023 Application project
//
// SPDX-FileCopyrightText: 2023 Stanford University
//
// SPDX-License-Identifier: MIT
//
```

File: CS342MemberViews/Sources/BrianPark/DescriptionView.swift [hunk 10]
```diff
@@ -0,0 +1,40 @@
+//
+//  File.swift
+//  
+//
+//  Created by Jeongwoo Park on 2023/01/24.
+//
```
Comment (vishnuravi) [inline CS342MemberViews/Sources/BrianPark/DescriptionView.swift]: There should be a REUSE header here, such as:

```swift
//
// This source file is part of the CS342 2023 Application project
//
// SPDX-FileCopyrightText: 2023 Stanford University
//
// SPDX-License-Identifier: MIT
//
```

**Comment by brian-jw-park** [comment]
Hi Vishnu! 

I've made the suggested edits; could you let me know if you receive them properly? I was getting an error saying the local repository was out of date so had to pull from the branch and then push the changes.

File: CS342MemberViews/Sources/BrianPark/DescriptionView.swift (hunk 10 above)
Comment (brian-jw-park) [reply CS342MemberViews/Sources/BrianPark/DescriptionView.swift]: @vishnuravi Hi Vishnu! I thought I properly committed my new changes but I'm not sure if I did. On my XCode when i try to push the new changes, I don't get an option for a location to push and when I try to commit, it says my files are now outdated? So I'm not sure if you see the updates or not. Thanks!

**Review by vishnuravi — APPROVED** [review]
Thank you Brian! Great job!
